schema/codelists/partyRole.csv

```diff
@@ -10,3 +10,12 @@ payee,Payee,A party in receipt of a payment from a transaction.,
 reviewBody,Review body,A party responsible for the review of this procurement process. This party often has a role in any challenges made to the contract award.,
 interestedParty,Interested party,"A party that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",
 contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting process.),
+privateParty,Private party,"The counter party of the procuring authority in the PPP contract. A private entity which has been granted the contract to construct and operate a government asset, and which is usually created under the form of a Special Purpose Vehicle or SPV.",
+leadBank,Lead bank,A lead bank is a bank that oversees the arrangement of a loan syndication,
+lender,Lender,"The APMG PPP Certification Program defines lenders as ""Institutions that provide lending to the project: mainly banks and institutional investors through project bonds""",
```

Comment:
Maybe we do want a source column, so that we can just use the straight definition, and reference the source by URL in the Source column only.

Reply:
Sourcing is always a good idea, but we must think of how it will look once displayed in a 700px-wide table (which is not wide). That would look good with a [Source](https://ppp-certification.com/ppp-certification-guide/glossary) link in the source column. Or just the domain name: [ppp-certification.com](https://ppp-certification.com/ppp-certification-guide/glossary).

Reply:
I add the Source column in partyRole.csv, and @jpmckinney takes care of making the table look good :)

Reply:
Once you add it back, you can check how it looks here: https://standard.open-contracting.org/staging/1157_addPPPCodes/en/schema/codelists/#party-role